ete3/evol/model.py

```diff
@@ -145,7 +145,7 @@ def __check_marks(self):
         checks if tree is marked and if model allows marks.
         fill up branches dict with marks
         """
-        has_mark = any([n.mark for n in self._tree.iter_descendants()])
+        has_mark = any(n.mark for n in self._tree.iter_descendants())
```

Comment:
I never used this syntax before. I guess nothing wrong with it. Should be faster based in iterations. @fransua ?